spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/dao/JdbcTaskExecutionDao.java

```diff
@@ -139,4 +258,45 @@ private void insertParameter(String executionId, String param) {
 		jdbcTemplate.update(getQuery(CREATE_TASK_PARAMETER), args, argTypes);
 	}
 
+	private List<String> getTaskParameters(String executionId){
```

Comment:
We should consider providing a more rich parameters object here instead of just a list of strings...